### DISCUSSION TITLE
課題04（チャレンジ問題）についての質問

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#16] [Q&A]
https://github.com/nsc-kiwamu/traning-takeda/commit/bf7f76a720fc9d35fdc6788c84008fb52cf145a1#commitcomment-77801931

↑上記のリンク先に質問内容を記載させていただいております。

**Comment by nsc-kiwamu** [comment]
https://github.com/nsc-kiwamu/traning-takeda/pull/8#discussion_r914815816
